CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.1.0
+
+- Initial release.
```

Comment:
Update with a basic description of the initial features

Reply:
ack